src/include/ndpi_typedefs.h

```diff
@@ -76,6 +76,8 @@ typedef enum {
   NDPI_TLS_NOT_CARRYING_HTTPS,
   NDPI_SUSPICIOUS_DGA_DOMAIN,
   NDPI_MALFORMED_PACKET,
+  NDPI_SSH_OBSOLETE_CLIENT_SIGNATURE,
+  NDPI_SSH_OBSOLETE_SERVER_SIGNATURE,
```

Comment:
Please read the comment at the beginning of this enumeration: you should update some other stuff, too